### PR TITLE
Improve class comments

### PR DESCRIPTION
**Pull request by strawberriesandcheese**

### CODE REVIEW

File: packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state.
\ No newline at end of file
+A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state.
+
+Instance Variables
+	changeRecord			represented ChangeRecord as actual Squeak ChangeRecord
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md:4]:
```suggestion
	changeRecord			actual ChangeRecord wrapped by CRChangeListItem
```

File: packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md [hunk 2]
```diff
@@ -1 +1,5 @@
-A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state.
\ No newline at end of file
+A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state.
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md:1]:
```suggestion
A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state. It wraps a ChangeRecord of the Squeak image and can be handled as such, all messages not understood by the CRChangeListItem are forwarded to the wrapped ChangeRecord.
```

File: packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md [hunk 3]
```diff
@@ -1 +1,5 @@
...
+
+Instance Variables
+	changeRecord			represented ChangeRecord as actual Squeak ChangeRecord
+	isPicked					wether a specific represented ChangeRecord is picked for loading
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md:5]:
```suggestion
	isPicked					true if the wrapped ChangeRecord is picked for getting loaded into the image
```

File: packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md (hunk 2 above)
Comment (marcelgarus) [reply packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md]: "A CRChangeListItem is an item **in** the…"
It's a shame there's no suggestion for suggestions

File: packages/ChangesReloaded-Core.package/CRBrowser.class/README.md [hunk 4]
```diff
@@ -3,4 +3,5 @@ A CRBrowser is a tool to recover changes stored in the changes file back into th
 On initialization, the CRBrowser creates a ChangeList to read all wanted changes as instances of ChangeRecord from the changes file. After that, it turns the OrderedCollection of ChangeRecords into an OrderedCollection of CRChangeListItems. That makes it possible to store a selection state for each ChangeRecord. The ChangeList is not needed after this step anymore and gets deleted. The CRBrowser performs future operations directly on the OrderedCollection of CRChangeListItems.
 
 Instance Variables
-	changeListitems		OrderedCollection of CRChangeListItems
+	changeListItems					OrderedCollection of CRChangeListItems
+	changeListSelectionIndex		Index of the currently selected Item in the ChangeList for viewing in detail
```
Comment (marcelgarus) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/README.md:7]:
```suggestion
	changeListSelectionIndex		index of the currently selected item in the ChangeList for viewing in detail
```

File: packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state.
+A CRChangeListItem is an item of the CRBrowser's change list that represents a ChangeRecord and has a selection state. It wraps a ChangeRecord of the Squeak image and can be handled as such, all messages not understood by the CRChangeListItem are forwarded to the wrapped ChangeRecord.
```
Comment (marcelgarus) [inline packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md:1]:
```suggestion
A CRChangeListItem is an item in the CRBrowser's change list that represents a ChangeRecord and has a selection state. It wraps a ChangeRecord of the Squeak image and can be handled as such, all messages not understood by the CRChangeListItem are forwarded to the wrapped ChangeRecord.
```

**Pull request merged**